src/main/java/betteradvancements/util/CriterionColumn.java

```diff
@@ -0,0 +1,13 @@
+package betteradvancements.util;
+
+import java.util.List;
+
+public class CriterionColumn {

```

Comment:
This one can be an inner class of `CriterionGrid`, but call it just `Column` so it will be `CriterionGrid.Column`.

Reply:
Makes sense.